src/router/index.js

```diff
@@ -10,14 +11,20 @@ const routes = [
     name: 'Home',
     component: Home
   },
+  {
+    path: '/results',
+    name: 'Results',
+    component: Results
```

Comment:
I love how Vue Router is set up this way.  Makes rendering the components a lot easier, and it is very organized!

Reply:
I agree! For small projects I like React Router just fine, but the way Vue does router seems a lot more scalable.